entity-service-change-event-generator/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ dependencies {
     }
   }
 
-  runtimeOnly("io.confluent:kafka-protobuf-serializer:6.0.1")
+  runtimeOnly("io.confluent:kafka-protobuf-serializer:7.4.0")
```

Comment:
Here, we should use the kafka bom. That should look something like

```
runtimeOnly("io.confluent:kafka-protobuf-serializer")

implementation(platform("org.hypertrace.core.kafkastreams.framework:kafka-bom:0.2.10")
```

Let me try to update a repo to get an example.

Reply:
> > What should we do for vulnerability in `snappy-java-1.1.8.4.jar`?

Also the kafka bom will fix this